### DISCUSSION TITLE
Resolve skin resources to osu!classic for older skins

### DISCUSSION THREAD

**Discussion by facespkz** [ppy/osu#13242] [Ideas]
As of writing, the current build of osu! loads all skins with lazer assets as a fallback. This can mess with some older skins where classic assets are expected, so skins with Version ≤ 2.x should use them instead of lazer.

**Comment by frenzibyte** [comment]
This is a known issue, and it affects all skin versions, not only "≤ 2.x", tracked here https://github.com/ppy/osu/issues/8019.
